src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcngEntityCommand.php

```diff
@@ -491,19 +491,19 @@ public function setSecret($secret)
      */
     public function getRedirectUrls()
     {
-        if (!is_array($this->redirectUrls)) {
-            return [];
-        }
```

Comment:
By removing this, you excavated a somewhat intricate buggy behavior. 

1. WebTests are unable to handle form interactions with the Redirect URLs as they are JavaScript based and our curl based test browser is not capable of JS interactions.
2. By removing the !is_array construction, the correct: 'at least one redirect url is required' form error is triggered on the form. This halts execution of the test.
3. Previously we would return an empty array, this somehow passed validation. And the web test was satisfied. 
4. In dev/prod environments this behavior would trigger an error in Manage. As it does not allow empty or unset redirect urls.

Hope this makes some sense.. 

Possible solutions include:
- to add a no-js solution for the redirect urls
- move this test to the cypress test suite
- implement real browser support into the web-tests (should be easy enough using [Panther](https://github.com/symfony/panther))
- insert your genius solutions here..